### ISSUE TITLE
ogrfdw doesn't compile against 9.5beta1

### ISSUE THREAD

**Issue by robe2** [pramsey/pgsql-ogr-fdw#40]
Just tried building pgsql-ogr-fdw against recently released PostgreSQL 9.5beta1, and sadly it doesn't even get out of the compile gate.

```

ogr_fdw.c: In function 'ogr_fdw_validator':
ogr_fdw.c:317:30: warning: variable 'layer' set but not used [-Wunused-but-set-variable]
  const char *source = NULL, *layer = NULL, *driver = NULL;
                              ^
ogr_fdw.c: In function 'ogrGetForeignPlan':
ogr_fdw.c:590:26: error: too few arguments to function 'make_foreignscan'
                          fdw_private);
                          ^
In file included from ogr_fdw.h:42:0,
                 from ogr_fdw.c:29:
e:/jenkins/postgresql/rel/pg9.5w64gcc48/include/server/optimizer/planmain.h:46:21: note: declared here
 extern ForeignScan *make_foreignscan(List *qptlist, List *qpqual,
                     ^
ogr_fdw.c:591:1: warning: control reaches end of non-void function [-Wreturn-type]
 }
 ^
```

**Comment by robe2** [comment]
I have added fix for 9.5beta1 to my original pull request.  I got the idea from https://gist.github.com/mc-soi/9f31c6d62e85292f8ff5

**Comment by robe2** [comment]
was in patch.
